ui/apps/everest/.e2e/release/pmm-k8s-metrics.e2e.ts

```diff
@@ -16,7 +16,7 @@
 import { test } from '@playwright/test';
 import { checkK8sMetrics } from '@e2e/utils/monitoring-instance';
 
-test(`Check PMM K8s metrics`, async () => {
+test(`T31 - Check PMM K8s metrics`, async () => {
```

Comment:
This test uses a hardcoded Zephyr `T31`  instead of fetching from a centralised zephyrMap object. I think we should keep consistency with the other files as well: ` test(`${zephyrId} - `